src/main/java/com/volmit/iris/util/decree/annotations/Param.java

```diff
@@ -49,7 +49,7 @@
      * Which indicates the variable MUST be defined by the person running the command.<br>
      * If you define this, the variable automatically becomes non-required, but can still be set.
      */
-    String defaultValue() default "";
+    String defaultValue() default REQUIRED;
```

Comment:
i explained this in vc

Reply:
See above

Reply:
No actually this is resolved. It's using "" again with .isEmpty()